AWSCore/Service/AWSService.m

```diff
@@ -253,6 +253,7 @@ - (id)copyWithZone:(NSZone *)zone {
 static NSString *const AWSRegionNameEUCentral1 = @"eu-central-1";
 static NSString *const AWSRegionNameEUCentral2 = @"eu-central-2";
 static NSString *const AWSRegionNameEUNorth1 = @"eu-north-1";
+static NSString *const AWSRegionNameILCentral1 = @"il-central-1";
```

Comment:
super low priority: if there's no ordering, perhaps add it to the last of the list?

Reply:
since the AWSServiceEnum new case is also at the end